charts/pulsar-admin-console/templates/tests/run-pulsar.yaml

```diff
@@ -0,0 +1,46 @@
+#
+#  Copyright 2022 DataStax, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+
+{{- if .Values.enableTests }}
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "pulsar"
+  labels: 
+    app: pulsar
+spec:
+  containers:
+  - name: "standalone"
+    image: "apachepulsar/pulsar:2.11.0"
```

Comment:
Would it make sense to expose this version in the values file to test different versions?

Reply:
done